jupyterhub/objects.py

```diff
@@ -149,6 +149,15 @@ class Hub(Server):
 
     cookie_name = 'jupyter-hub-token'
 
+    pending_spawns = Integer(
```

Comment:
There's something about the variable name `pending_spawns` that doesn't feel quite right. `spawns_pending` seems to be a bit better or something else without `pending` perhaps.